analyzers/tests/SonarAnalyzer.Test/TestCases/CloudNative/AzureFunctionsReuseClients.HttpClient.CSharp9.cs

```diff
@@ -19,7 +19,7 @@ public static void DifferentAssigments()
             field ??= new HttpClient();                       // Compliant
 
             var local = default(HttpClient);
-            local ??= new HttpClient();                       // Noncompliant
+            local ??= new HttpClient();                       // Compliant
```

Comment:
This is compliant, I verified with Martin that implemented the rule.

Reply:
It's conditional code in a way—if it's null, then create a new client. In real cases, this might be a valid condition under which a user needs to create a new client. We don't want to raise FPs.